domain/schema/schema_test.go

```diff
@@ -859,6 +864,17 @@ func (s *schemaSuite) TestModelChangeLogTriggersForSecretTables(c *gc.C) {
 	s.assertChangeLogCount(c, 2, tableSecretReference, 1)
 	s.assertChangeLogCount(c, 4, tableSecretReference, 1)
 
+	// secret_deleted_value_ref table triggers.
+	deletedRvisionUUID := utils.MustNewUUID().String()
+	backendUUIDUUID := utils.MustNewUUID().String()
+	s.assertChangeLogCount(c, 1, tableSecretDeletedValueRef, 0)
+	s.assertChangeLogCount(c, 2, tableSecretDeletedValueRef, 0)
+	s.assertChangeLogCount(c, 4, tableSecretDeletedValueRef, 0)
+
+	// We only care about inserts into this table.
+	s.assertExecSQL(c, `INSERT INTO secret_deleted_value_ref (revision_uuid, backend_uuid, revision_id) VALUES (?, ?, 1);`, deletedRvisionUUID, backendUUIDUUID, "rev")
+	s.assertChangeLogCount(c, 1, tableSecretDeletedValueRef, 1)
```

Comment:
If this test gets any bigger, we should split it out. It's getting a bit bonkers.